clients/admin-ui/src/features/system/form.ts

```diff
@@ -1,8 +1,10 @@
+import { CustomFieldsFormValues } from "~/features/common/custom-fields";
 import { DEFAULT_ORGANIZATION_FIDES_KEY } from "~/features/organization";
 import { DataProtectionImpactAssessment, System } from "~/types/api";
 
 export interface FormValues
-  extends Omit<System, "data_protection_impact_assessment"> {
+  extends Omit<System, "data_protection_impact_assessment">,
+    CustomFieldsFormValues {
```

Comment:
The benefit to putting the `CustomFieldsFormValues` into its own type is that these forms can then be explicit that they these properties as well. Previously the system and the taxonomy could end up including the custom fields in the JSON for an endpoint that doesn't expect it.